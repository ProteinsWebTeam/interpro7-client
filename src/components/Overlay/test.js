// @flow
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import { Overlay } from '.';

const renderer = new ShallowRenderer();

describe('<Overlay />', () => {
  test('should render as visible', () => {
    renderer.render(<Overlay visible closeEverything={() => {}} />);
    expect(renderer.getRenderOutput()).toMatchSnapshot();
  });

  test('should render as invisible', () => {
    renderer.render(<Overlay visible={false} closeEverything={() => {}} />);
    expect(renderer.getRenderOutput()).toMatchSnapshot();
  });
});
