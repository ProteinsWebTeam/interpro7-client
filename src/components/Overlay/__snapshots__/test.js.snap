// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Overlay /> should render as invisible 1`] = `
<div
  aria-hidden="true"
  className="overlay"
  onClick={[Function]}
  onKeyPress={[Function]}
/>
`;

exports[`<Overlay /> should render as visible 1`] = `
<div
  aria-hidden="true"
  className="overlay visible"
  onClick={[Function]}
  onKeyPress={[Function]}
/>
`;
